### ISSUE TITLE
After upgrading Terraform Provider (with Rancher 2.3.x), rancher2_catalog keeps trying to force new for version

### ISSUE THREAD

**Issue by Oats87** [rancher/terraform-provider-rancher2#298]
After upgrading to the latest available Terraform provider, my existing Terraform keeps trying to replace my rancher2_catalog (and subsequent rancher2_app) definitions because the "helm_v2" version is not set.

In order to workaround this, I had to set:
```
 lifecycle {
    ignore_changes = [
      version
    ]
  }
```

**Comment by rawmind0** [comment]
Submitted PR #296 to fix the issue

**Comment by rawmind0** [comment]
PR included on release v1.8.2. Please reopen issue if needed.
